CFC_WebApp/main/gmap_display.py

```diff
@@ -148,7 +148,7 @@ def drawSection(section, option, gmap, Color = 'default'):
                 # coordinates are in GeoJSON format, ie lng, lat
                 gmap.addpoint(coordinate[1], coordinate[0], color)
         if option == PATH or option == ALL:
-            gmap.addpath(path, color, _id)
```

Comment:
I think I figured it out. In utils/plot_trip.py, it was importing pygmaps instead of pygmaps_modified, so the modified function was never being called. I'm fixing it.